engine-precompiles/src/xcc.rs

```diff
@@ -31,7 +31,7 @@ pub mod costs {
     ///
     /// This process is done in the `test_xcc_eth_gas_cost` test in
     /// `engine-tests/src/tests/xcc.rs`.
-    pub const CROSS_CONTRACT_CALL_BASE: EthGas = EthGas::new(323_000);
+    pub const CROSS_CONTRACT_CALL_BASE: EthGas = EthGas::new(343_650);
```

Comment:
I guess this is what you were talking about. It's a little concerning that a dependency update would so significantly increase gas cost.

Reply:
Yes. Exactly.

Reply:
Yes, raising this in particular is bad because it breaks backwards compatibility (the same transaction prior to this change will cost less than after this change). But XCC was only just released on mainnet so I think it's ok to change it now.